### PR TITLE
(german) translated weapon and swarmopedia strings

### PR DESCRIPTION
**Pull request by DukeDukeEm**

### CODE REVIEW

File: resource/reactivedrop_german.txt [hunk 1]
```diff
@@ -743,8 +743,8 @@
 "[english]asw_weapon_flashlight"		"Flashlight"
 "asw_weaponl_flashlight"		"Taktisches Licht"
 "[english]asw_weaponl_flashlight"		"Flashlight Attachment"
-		"asw_weapon_flashlight_attributes"		"Tactical"
-		"[english]asw_weapon_flashlight_attributes"		"Tactical"
+"asw_weapon_flashlight_attributes"		"Passiv"
```
Comment (BenLubar) [inline resource/reactivedrop_german.txt:746]: Taktisch?

Comment (DukeDukeEm) [reply resource/reactivedrop_german.txt:746]: Yea, but tactical doesn't really mean anything in this context. Passive tells the user something tactical about the item.

Comment (BenLubar) [reply resource/reactivedrop_german.txt:746]: It's a reference to https://youtu.be/urMHvdLz0Jg?t=246

Comment (DukeDukeEm) [reply resource/reactivedrop_german.txt:746]: image

Comment (BenLubar) [reply resource/reactivedrop_german.txt:746]: Ah, you moved the reference to a different string.

All good.

**Pull request merged**